samples/SocketsSample/SocketsSample.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\dependencies.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PackageTargetFallback Condition=" '$(TargetFrameworks)' == 'netcoreapp2.0' ">$(PackageTargetFallback);portable-net45+win8+wp8+wpa81</PackageTargetFallback>
```

Comment:
We might not need it  which is why I wanted @mikaelm12 to try removing it first. But even if we do include it, it should be TargetFramework singular not TargetFrameworks plural.